golangci.yaml

```
@@ -0,0 +1,32 @@
+run:
+  timeout: 6m
+
+linters:
+  disable-all: true
+  enable:
+    - bodyclose
+    - deadcode
+    - dogsled
+    - gocritic
+    - goimports
+    - golint
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    - maligned
+    - misspell
+    - nakedret
```

```suggestion
    - nakedret
    - revive
```

Replaces `golint`, see above.